pages/profile/[userId].tsx

```diff
@@ -281,6 +300,15 @@ export default function InternalProfile({
         userProfileData={userProfileData}
         isEditable={!isExternal}
       />
+      <div className="flex justify-center">
+        {accountabilityLoading ? (
+          <LoadingComponent />
+        ) : accountabilityError ? (
+          <p>An error occurred while fetching data.</p>
+        ) : (
+          <AccountabilityHeatmap entries={data?.accountability ?? []} />
```

Comment:
Can we left align this component so it looks aligned to the profile header and other section components? It looks off when it's the only one centered. 

Make sure it looks good on mobile